### DISCUSSION TITLE
No author previous degree

### DISCUSSION THREAD

**Discussion by joaopio-unigit** [joaomlourenco/novathesis#446] [Q&A] [answered]
Hello everyone!!
I was updating the cover of my master's thesis and realized that I can't comment the author's previous degree to make it disappear from the cover, something that I can do with the sub-title for example.
Since I'm doing an integrated master's I don't feel like I have a previous degree.

I don't know much of Latex so I can't make the change and make a pull request so I leave it here as a note so that someone can suggest a solution and maybe update the project.

Thank you all!!

**Comment by joaomlourenco** [comment, marked as answer]
@joaopio-unigit 
May I ask what is your School/University?
Because, if it is FCT-NOVA, the line with previous degree is not optional. 
If it is some other School, check the rules first.  Than, it if it really optional, let me know andI'll help you.
